data_pipeline/data_processing2.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Data Processing
+# ## Data manipulation - Data Cleansing and ETL
+# ### From Excel files to one master csv format file and MySQL ingestion
+# 
+#     Author:  Jacky Hung - March 2020
+#     - Incorporated Helen's de-dup script.
+#     - Enhanced with more data cleansing and ETL
+#    
+# Pre-requisite for running this notebook.
+#     python3
+#     python packages:  numpy, pandas, seaborn, matplotlib, pymysql, sqlalchemy
+#
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import time
+import seaborn as sns
+
+ods = pd.read_csv('460_460A.csv', 
+#                   index_col=0,
+                   dtype={'Agent_Nam F': str,
+                          'Sup_Opp_Cd': str,
+                          'G_From_E_F': str,
+                          'XRef_Sch Nm': str},
+                   parse_dates=[3,4,5,6])
+
+ods.shape
+
+
+# In[2]:
```

Comment:
Nit (not required to fix): The #In[2] and two line empty spacers make this somewhat hard to read, could they be left out/replaced with single empty line spacers?